pkg/config/dynamicconfig.go

```diff
@@ -62,13 +95,25 @@ func NewConfigData(rclient kubernetes.Interface, cmInformer informers.ConfigMapI
 		cmSycned: cmInformer.Informer().HasSynced,
 		log:      log,
 	}
+	cd.restrictDevelopmentUsername = []string{"minikube-user", "kubernetes-admin"}
+
```

Comment:
Shouldn't the initial value be taken from the flag? Can you update the doc on how these flags / configmap working accordingly? We can make it a separate PR.

Reply:
ok